### ISSUE TITLE
Proposal: Split Browse page Submissions into Visual / Literary / Multimedia

### ISSUE THREAD

**Issue by huntertur** [Weasyl/weasyl#1468]
A was chatting with a writing friend earlier. They were telling me about the importance of websites displaying stories separately from art to better help with visibility, and mentioned some art websites that have done this. They specifically pointed out the current Browse page as one of the reasons they don't use Weasyl.

I noticed the Weasyl Browse page (`/search` with no parameters) is partially split: of the five content types, there are three. Characters and Journals are separated, but Visual, Literary, and Multimedia submissions compete for the general Submissions category. Since Literary and Multimedia submissions get uploaded less frequently than Visual -- as is the same with Characters and Journals -- I think it would make sense to split these categories out too. One potential downside could be performance concerns, but I haven't dug into the existing search/browse code yet.

What do you all think? Do you think this would be beneficial?

**Comment by charmander** [comment]
Yes! The current Browse page is pretty much a worse version of the homepage, unless you want to see journals. (Selecting “Browse Everything” from the homepage actually shows you *less* of the same stuff[^1] above the fold, thanks to the redundant search bar.)

(Of course, Weasyl is just really bad for writing in general at this point, but I’ve been working on that too!)

[^1]: except for submissions that were excluded by the artist occurrence limit

**Comment by huntertur** [comment]
RE: seeing less of the same stuff: I can look at removing the redundant search bar; would you want the Browse Content heading removed too, or kept?

Also: although #1423 is concerned about the homepage and this issue is concerned about the Browse page, do you have any concerns on which change gets merged first? I don't want to step on your toes with any test changes, and your change looks ready to go while I haven't touched the implementation work on this yet.

**Comment by charmander** [comment]
Don’t worry about #1423 – some people didn’t like it, so it’s a low-priority rework. And I’m not afraid of merge conflicts 🙂
